### PR TITLE
[vstest]: Add test_port_config.py which include breakout port test.

### PR DESCRIPTION
**Pull request by chiourung**
[pytests] Add test_port_config.py which include breakout port test.

It needs to modify sai_create_port for virtual switch.

**What I did**
add test case for port breakout

**Why I did it**
make sure the orchagent is correct for port breakout

**How I verified it**
run the pytest

**Details if related**
It also needs to modify sai_create_port for virtual switch (Azure/sonic-sairedis#453)

Signed-off-by: chiourung_huang <chiourung_huang@edge-core.com>

### CODE REVIEW

**Comment by chiourung** [comment]
The test_port_breakout needs to merge the commit in sonic-sairedis. (Modify sai_create_port for virtual switch #453)
After merge #453, it can pass the test_port_breakout.

**Comment by chiourung** [comment]
retest this please

**Comment by chiourung** [comment]
retest this please

**Comment by chiourung** [comment]
retest this please

**Comment by chiourung** [comment]
retest this please

**Comment by chiourung** [comment]
@lguohan , May you help to review? Thanks.

File: tests/test_port_config.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        for i in range(0,4):
+            port_name = 'Ethernet{0}'.format(eth+i)
+            port_oid = self.getPortOid(dvs, port_name)
+            port_tbl = swsscommon.Table(asic_db, 'ASIC_STATE:SAI_OBJECT_TYPE_PORT:{0}'.format(port_oid))
+
+            for k in port_tbl.get('')[1]:
+                if k[0] == "SAI_PORT_ATTR_HW_LANE_LIST":
+                    hw_lane_value = k[1]
+
+            assert hw_lane_value == "1:%s" % (LANES_L[i])
```
Comment (stcheng) [inline tests/test_port_config.py]: E           UnboundLocalError: local variable 'hw_lane_value' referenced before assignment

**Comment by lguohan** [comment]
retest this please

File: tests/test_port_config.py (hunk 1 above)
Comment (chiourung) [reply tests/test_port_config.py]: I have modified code to assign value for hw_lane_list before reference. Do you have any comment?

Comment (chiourung) [reply tests/test_port_config.py]: @stcheng  I have modified code to assign value for hw_lane_list before reference. Do you have any comment?

**Comment by stcheng** [comment]
retest this please

**Comment by chiourung** [comment]
retest this please

**Comment by stcheng** [comment]
Retest this please

**Comment by chiourung** [comment]
Retest this please

**Comment by lguohan** [comment]
retest this please

**Pull request merged**